### PR TITLE
[xc-admin] rework pyth hook

### PR DESCRIPTION
**Pull request by guibescos**
Took a deep dive in the hook and deeply simplified. 
It querries all the program accounts and then build the tree mapping-product-price.
This gives the most accurate picture of the on-chain state.

### CODE REVIEW

File: governance/xc-admin/packages/xc-admin-frontend/hooks/usePyth.ts [hunk 1]
```diff
@@ -1,201 +1,74 @@
...
-            }
-          })
-          .filter((_) => _) as AccountInfo<Buffer>[]
-    )
-    .flat()
-  return { keys, array }
-}
-
-export const ORACLE_PUBLIC_KEYS: { [key: string]: string } = {
+export const MAPPING_ACCOUNT_KEYS: { [key: string]: string } = {
```
Comment (cctdaniel) [inline governance/xc-admin/packages/xc-admin-frontend/hooks/usePyth.ts:20]: should this be exposed in pyth-client-js instead and imported here?

File: governance/xc-admin/packages/xc-admin-frontend/hooks/usePyth.ts [hunk 2]
```diff
@@ -204,160 +77,144 @@ const usePyth = (
...
-              ...allProductAccountKeys,
-              ...productAccountKeys,
-            ]
-            anotherMappingAccount = nextMappingAccount
+        const allPythAccounts = await connection.getProgramAccounts(
+          getPythProgramKeyForCluster(cluster)
+        )
+        const priceRawConfigs: { [key: string]: PriceRawConfig } = {}
+
+        console.log('LENGHT BEFORE FIRST PASS : ', allPythAccounts.length)
```
Comment (cctdaniel) [inline governance/xc-admin/packages/xc-admin-frontend/hooks/usePyth.ts:86]: maybe remove console.log statements?

**Review by cctdaniel — APPROVED** [review]
🙇🏻‍♂️

File: governance/xc-admin/packages/xc-admin-frontend/hooks/usePyth.ts (hunk 1 above)
Comment (guibescos) [reply governance/xc-admin/packages/xc-admin-frontend/hooks/usePyth.ts:20]: I think this currently unused so wouldn't mind deleting it, but we could totally move it to pyth-client-js

File: governance/xc-admin/packages/xc-admin-frontend/hooks/usePyth.ts (hunk 2 above)
Comment (guibescos) [reply governance/xc-admin/packages/xc-admin-frontend/hooks/usePyth.ts:86]: Stale after merging your PR

File: governance/xc-admin/packages/xc-admin-frontend/hooks/usePyth.ts (hunk 1 above)
Comment (cctdaniel) [reply governance/xc-admin/packages/xc-admin-frontend/hooks/usePyth.ts:20]: let's delete it then

**Pull request merged**